spark-common/src/main/resources/com/k15t/spark/content_iframe_wrapper.vm

```diff
@@ -1,20 +1,21 @@
 ## appending WithHtml to variable name instructs Velocity not to add html escaping (like & -> &amp;)
 ## which would otherwise broke the js embedded this way pretty badly...
-#set($iframeResizerJsWithHtml = $iframeResizerJs)
+#set($sparkJsWithHtml = $sparkJs)
 <script>
-    ${iframeResizerJsWithHtml}
+    ${sparkJsWithHtml}
 </script>
-<iframe id="${iframeId}" scrolling="no" style="width: 100%; border: none;" src="${iframeSrc}"></iframe>
+<div id="${iframeId}-wrapper"></div>
 <script>
-    if (iFrameResize) {
-        var iframeEl = document.getElementById("${iframeId}");
+    AJS.toInit(function() {
```

Comment:
We should wrap this into an IIFE in order to avoid calling getSpark() inside the (delayed) AJS.toInit callback function:

```
(function(SPARK) {
    AJS.toInit(function() {
        ...
        SPARK.apploader.loadApp(...)
        ...
    });
})(window.getSpark());